### PR TITLE
Remove 404'ing links

### PR DESCRIPTION
**Pull request by TanayParikh**
#### Summary

These references no longer resolve: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/addHitRegion

This PR…
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by teoli2003** [comment]
If we have deleted the `Canvas.addHitRegion` page, we should delete this one too.

**Comment by TanayParikh** [comment]
> If we have deleted the `Canvas.addHitRegion` page, we should delete this one too.

Updated.

**Comment by teoli2003** [comment]
Thank you! 

In order to _delete_ a page, you have to follow this procedure: https://github.com/mdn/content#deleting-a-document
We store meta documentation with editor names and we have a file storing redirects. This procedure will take care of it.

(Note we want a redirect from /Web/API/MouseEvent/region to /Web/API/MouseEvent to prevent 404.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Please use the correct procedure to delete a file.

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # FROM-URL	TO-URL
 /en-US/docs/-moz-locale-dir(ltr)	/en-US/docs/Web/CSS/:-moz-locale-dir(ltr)
 /en-US/docs/-moz-locale-dir(rtl)	/en-US/docs/Web/CSS/:-moz-locale-dir(rtl)
+/en-US/docs//Web/API/MouseEvent/region	/en-US/docs/web/api/mouseevent
```
Comment (teoli2003) [inline files/en-us/_redirects.txt:9]: Capitalization is wrong: /en-US/docs/Web/API/MouseEvent 

(same below)

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/_redirects.txt (hunk 1 above)
Comment (TanayParikh) [reply files/en-us/_redirects.txt]: Yep just noticed that as well, not sure what happened. Sorry about that, updated appropriately now.

**Review by teoli2003 — APPROVED** [review]
Thank you!

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**